Bootcamp2015.AmazingRace.Base/APIModels/LeaderboardValues.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bootcamp2015.AmazingRace.Base.APIModels
+{
+    public class LeaderboardValues
+    {
+        public string id;
+        public string name;
+        public IEnumerable<Team> teams;
+
+        public class Team
+        {
+            public string id;
+            public string name;
+            public string imageUri;
+            public string rank;
+            public string points;
+            public string latitute;
+            public string longitude;
```

Comment:
You can use properties with proper PascalCase for this and the Json Serializer is smart enough to map "Id" to "id".